translib/transformer/xfmr_intf.go

```diff
@@ -973,11 +1089,14 @@ var YangToDb_intf_subintfs_xfmr KeyXfmrYangToDb = func(inParams XfmrParams) (str
 
 	log.Info("YangToDb_intf_subintfs_xfmr: i32: %s", i32)
 
-	if i32 != 0 {
+	if idx != "0" && idx != "*" && idx != "" {
```

Comment:
`i32` is getting unused here. We can remove `i32` variable fetch code and variable itself, if not needed. Please test, GET/SET works now for subinterfaces index value != 0 now. Earlier with previous PR, it was throwing error "Subinterfaces not supported".